autode/hessians.py

```diff
@@ -400,13 +389,42 @@ def frequencies_proj(self) -> List[Frequency]:
                 "Could not calculate projected frequencies, must "
                 "have atoms set"
             )
-
         n_tr = self.n_tr  # Number of translational+rotational modes
-        lambdas = np.linalg.eigvalsh(self._proj_mass_weighted[n_tr:, n_tr:])
 
-        trans_rot_freqs = [Frequency(0.0) for _ in range(n_tr)]
+        H = self._proj_mass_weighted
+        norms = np.linalg.norm(H, axis=0)
+        max_norm = np.max(norms)
+        n_zeroed_modes = 0  # Number of modes that have been well projected out of the hessian
+        for norm in norms:
+            if norm / max_norm > 0.1 or n_zeroed_modes == n_tr:
+                break
+            else:
+                n_zeroed_modes += 1
+
+        if n_zeroed_modes != n_tr:
+            logger.warn(
+                f"Number of well zeroed eigenvectors of the hessian "
+                f"was [{n_zeroed_modes}] should be [{n_tr}]"
+            )
+
+        lambdas = np.linalg.eigvalsh(H[n_zeroed_modes:, n_zeroed_modes:])
+        trans_rot_freqs = [Frequency(0.0) for _ in range(n_zeroed_modes)]
         vib_freqs = self._eigenvalues_to_freqs(lambdas)
 
+        n_rotational_modes_in_vib_freqs = n_tr - n_zeroed_modes
```

Comment:
Same with this part - I think it is not required.

Reply:
@t-young31 Sorry, have been very busy the past few days (preparing poster for conference). I will have a look soon.

Reply:
no worries! hope the conference goes well 😄